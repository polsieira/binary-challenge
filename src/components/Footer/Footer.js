import React from 'react';
import './Footer.scss';

const Footer = () => {
  return (
    <div className='Footer'>
      <p>Footer</p>
    </div>
  );
};

export default Footer;