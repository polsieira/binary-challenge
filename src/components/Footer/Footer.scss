.Footer {
  background-color: white;
  bottom: 0;
  display: flex;
  justify-content: flex-end;
  left: 0;
  margin-top: 50px;
  padding: 20px;
  right: 0;

  
  img {
    height: 40px;
  }
}