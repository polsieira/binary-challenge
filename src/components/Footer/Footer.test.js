import Footer from './Footer'
import { shallow } from 'enzyme';
import React from 'react';


describe('Footer', () => {

  let wrapper;

  describe('Footer component', () => {
    beforeEach(() => {
      wrapper = shallow(<Footer />)
    })

    it('should match a snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });

  });

});

