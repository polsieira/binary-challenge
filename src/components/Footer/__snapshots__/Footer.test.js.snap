// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer component should match a snapshot 1`] = `
<div
  className="Footer"
>
  <img
    alt="special thanks to NASA"
    src="https://exoplanetarchive.ipac.caltech.edu/images/logos/NASA_small.jpg"
  />
  <img
    alt="special thanks to CalTech"
    src="https://exoplanetarchive.ipac.caltech.edu/images/logos/Caltech_LOGO-Orange_PMS165.jpg"
  />
</div>
`;
